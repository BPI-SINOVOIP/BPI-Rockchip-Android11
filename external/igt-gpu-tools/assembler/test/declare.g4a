.declare X1 Base=g99.0 ElementSize=1 SrcRegion=<8,8,1> DstRegion=<1> Type=F
.declare X1 Base=g123.4 ElementSize=4 SrcRegion=<8,8,1> DstRegion=<1> Type=F
add g0<1>:f X1 1.23:f;
add g0<1>:f X1 g1<8,8,1>:f;
add X1 g0<8,8,1>:f g1<8,8,1>:f;
