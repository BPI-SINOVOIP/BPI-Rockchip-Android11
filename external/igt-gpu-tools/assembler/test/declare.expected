   { 0x00e00040, 0x20007fbd, 0x008d0f64, 0x3f9d70a4 },
   { 0x00e00040, 0x200077bd, 0x008d0f64, 0x008d0020 },
   { 0x00e00040, 0x2f6477bd, 0x008d0000, 0x008d0020 },
